parm/streams.init_atmosphere

```diff
@@ -6,9 +6,24 @@
 
 <immutable_stream name="output"
                   type="output"
-                  filename_template="@output_stream@"
+                  filename_template="none.nc"
```

Comment:
Just curious. what is this change for?

Reply:
We will use the `output` stream for the next `jedi_ics` stream, but we don't want it to write out any files, so we set `output_interval="none"` for this stream.    
At the same time, we still need to give a unique name to `filename_template` as a placeholder. It can be anything. I just used `none.nc`.  You can understand `none.nc` means nothing, :)